test_utils/enterprise_report_choices.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""
+Enterprise Django application tests constants.
+"""
+
+LIMITED_REPORT_TYPES = {
+    'data_type': [
+        ['catalog', 'catalog'],
+        ['engagement', 'engagement'],
+        ['progress', 'progress_v3']
```

Comment:
it should be `['progress'_v3, 'progress']`,  as we can see the value of `day_of_week`.
(Apologies for yesterday comment, I said in opposite order yerterday)